packages/core/src/App/Containers/SetResidenceModal/set-residence-form.jsx

```diff
@@ -15,7 +15,7 @@ const SetResidenceForm = ({
     residence_list,
 }) => (
     <div className={`${class_prefix}__residence-selection`}>
-        {!!header_text && <p className={`${class_prefix}__heading`}>{header_text}</p>}
+        {!!header_text && <Text as='p' weight='bold' className={`${class_prefix}__heading`}>{header_text}</Text>}
```

Comment:
Question: since we add bold here, should we remove bold style from css as well or it' wasn't there already?

Reply:
the css file is account-signup and in __heading bold is already removed. Sara will create a card to handle it dynamically .